src/main/java/org/example/homework4/LeapYear.java

```diff
@@ -0,0 +1,24 @@
+package org.example.homework4;
+
+public class LeapYear {
+    public static void main(String[] args) {
+        int year = 8000;
+        boolean leapYear = isLeapYear(year);
+        if (leapYear)
+            System.out.println("The year " + year + " is leap");
+        else System.out.println("The year " + year + " is not leap");
+    }
+
+    public static boolean isLeapYear (int year) {
+        boolean leapYear;
+        if (year < 1 || year > 9999)
+            leapYear = false;
```

Comment:
This style looks better. And you don't need to create redundant `boolean leapYear`
```suggestion
            return false;
```